.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
           'EXPERIMENTAL_use',
```

Comment:
```
"capIsNewExceptionPattern": "^EXPERIMENTAL_"
```

https://eslint.org/docs/rules/new-cap#capisnewexceptionpattern

this will be nicer, since now we have older items that are still listed here, but no longer experimental

Reply:
https://github.com/algolia/instantsearch.js/pull/4581/commits/bf6d953fe87ce3a3fff66a6d20756d0fa5150788